helm_deploy/create-and-vary-a-licence/values.yaml

```diff
@@ -41,6 +41,7 @@ generic-service:
       SYSTEM_CLIENT_SECRET: "SYSTEM_CLIENT_SECRET"
       SESSION_SECRET: "SESSION_SECRET"
       TAG_MANAGER_CONTAINER_ID: "TAG_MANAGER_CONTAINER_ID"
+      LICENCE_WATERMARK: false
     elasticache-redis:
```

Comment:
Is this in the wrong place? These are secrets to read from the K8s secrets, so it will look for a secret called "false". I think it can just go into the `env:` section directly?

Reply:
Thanks, moved it now